mix.exs

```diff
@@ -32,6 +32,7 @@ defmodule Phoenix.Mixfile do
      {:poison, "~> 1.3"},
      {:redo, github: "heroku/redo", optional: true},
      {:poolboy, "~> 1.4.2", optional: true},
+     {:ecto, "~> 0.1", optional: true},
```

Comment:
This still should not be here. Can you please revert the changes to `Phoenix` mixfile? THank you!